components/workouts/CreateWorkout.jsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable react/prop-types, no-shadow */
```

Comment:
I think this is an important eslint check, I don't think we should disable this. i'll make a comment on how you can fix it